dom.bs

```diff
@@ -4363,6 +4363,26 @@ dom-Range-extractContents, dom-Range-cloneContents -->
  <a for=tree>children</a> of <var>node</var> and append them to <var>copy</var>, with
  <var>document</var> as specified and the <i>clone children flag</i> being set.
 
+ <li>
+  <p>If <var>node</var> is a <a for=Element>shadow host</a> whose <a for=/>shadow root</a>'s
+  <a for=ShadowRoot>clonable</a> is true:
+
+  <ol>
+   <li><p>Run <a>attach a shadow root</a> with <var>copy</var>, <var>node</var>'s <a for=/>shadow
+   root</a>'s <a for=ShadowRoot>mode</a>, true, <var>node</var>'s <a for=/>shadow root</a>'s
+   <a>delegates focus</a>.</p></li>
```

Comment:
No need for closing tags in DOM.

Reply:
I would hope that the file has internal consistency, but perhaps there are some errors here and there? In general they are disallowed, yes.

Reply:
I meant consistency between WHATWG/html and WHATWG/dom.